src/server/generic_family_test.cc

```diff
@@ -478,7 +478,17 @@ TEST_F(GenericFamilyTest, Sort) {
 
   Run({"set", "foo", "bar"});
   ASSERT_THAT(Run({"sort", "foo"}), ErrArg("WRONGTYPE "));
-  ;
+
+  Run({"rpush", "list-3", ""});
+  ASSERT_THAT(Run({"sort", "list-3"}), "");
+
+  Run({"rpush", "list-3", "2", "0", "", "-0.14", "0.12", "-0", "-123123", "7654"});
```

Comment:
Can you plz add nother test with a different list that contains a string ? For example all the elements above + "kostas". Does df reject now `SORT list` ? If it doesn't we have a bug

Reply:
we already has such test
  Run({"lpush", "list-2", "NOTADOUBLE"});
  ASSERT_THAT(Run({"sort", "list-2"}), ErrArg("One or more scores can't be converted into double"));